sphinxcontrib/jsontable/directives.py

```
@@ -653,10 +712,86 @@ def _load_json_data(self) -> JsonData:
+                    headers = excel_data["headers"]
+                    json_data = []
+                    for row in excel_data["data"]:
+                        row_obj = {}
+                        for i, value in enumerate(row):
+                            if i < len(headers):
+                                row_obj[str(headers[i])] = value
+                        json_data.append(row_obj)
+                    return json_data
+                else:
+                    # ヘッダーなし: 2D配列形式
+                    return excel_data["data"]
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)